### PR TITLE
bumping up to ForgeRock 6.5.0.1

### PR DESCRIPTION
**Pull request by thitu**
- updating `master` branch to ForgeRock `6.5.0.1`
- the previous master support ing FR 5.5 is at `master-5.5`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -14,30 +23,19 @@ buildscript {
 }
 
 group "com.trusona.integration"
-version "1.3.3-SNAPSHOT"
+version "1.4.0-SNAPSHOT"
```
Comment (tompiscitell) [inline build.gradle]: Should this be a major version bump to coincide with the FR major version bump?

File: src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java [hunk 2]
```diff
@@ -23,10 +23,10 @@ public TrucodeAuthenticator(TrusonaApi trusona) {
     Principal principal = null;
 
     TrusonaficationResult result = trusona.createTrusonafication(Trusonafication.essential()
-      .truCode(trucodeId)
-      .action(action)
-      .resource(resource)
-      .build());
+                                                                   .truCode(trucodeId)
```
Comment (tompiscitell) [inline src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java]: Did you use a linter for this? I don't think chained methods should be justified with the first opening paren. Thats usually done for method args.

Comment (thitu) [reply src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java]: I did use whatever the default is in in intellij

File: build.gradle (hunk 1 above)
Comment (thitu) [reply build.gradle]: like `2.0.0`? I can bump it higher

File: src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java (hunk 2 above)
Comment (thitu) [reply src/main/java/com/trusona/forgerock/auth/TrucodeAuthenticator.java]: my intellij config is set to [this](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)

**Pull request merged**